Local Pods/GitHubAPI/GitHubAPI.podspec

```diff
@@ -9,6 +9,6 @@ Pod::Spec.new do |spec|
   spec.source_files = 'GitHubAPI/*.swift'
   spec.ios.deployment_target     = '11.0'
   spec.watchos.deployment_target = '3.0'
-  spec.dependency 'Alamofire', '~> 4.4.0'
+  spec.dependency 'Alamofire', '~> 4.8'
```

Comment:
I was a bit confused that we need to update this in the `podspec` _and_ the [Podfile](https://github.com/GitHawkApp/GitHawk/pull/2650/files#diff-3083edf0ef607bdec8f7c6a7c2ee663eR8) 😅

Feels like one of the two might not be necessary?